src/components/Resource/Forms/ModelVersionUpsertForm.tsx

```diff
@@ -169,6 +172,7 @@ export function ModelVersionUpsertForm({ model, version, children, onSubmit }: P
   }: Schema) => {
     const schemaResult = querySchema.safeParse(router.query);
     const templateId = schemaResult.success ? schemaResult.data.templateId : undefined;
+    const bountyId = schemaResult.success ? schemaResult.data.bountyId : undefined;
 
     if (isDirty || !version?.id || templateId) {
```

Comment:
Should we add `bountyId` in this condition?
```suggestion
    if (isDirty || !version?.id || templateId || bountyId) {
```